modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1011,14 +1011,14 @@
 //What happens when the user clicks on datum
 /datum/action/item_action/hands_free/necklace_of_the_forsaken/Trigger()
 	var/obj/item/clothing/neck/necklace/necklace_of_the_forsaken/MM = target
-	if(MM.numUses == 0)//skip if it has already been used up
+	if(MM.numUses <= 0)//skip if it has already been used up
 		return
 	if(!MM.active_owner)//apply bind if there is no active owner
-		if(ishuman(owner))
+		if(iscarbon(owner))
```

Comment:
I feel like these twos can be switched perhaps?

Reply:
Is carbon should be better.